### ISSUE TITLE
Accessibility Concerns

### ISSUE THREAD

**Issue by wesleyboar** [MaayanLab/cfde-wheel#9]
## Overview

I notice UX that I think are accessibility issues:

1. **Links are Inside Buttons**
2. **Hover State Makes Logo Unclear**
3. **No Hover State for Pentagon Pieces**

### 1. Links are Inside Buttons

Interaction should be a link or a button, **not both**. Both is semantically unclear, thus confusing for keyboards and assistive technology.

1. User navigates via `tab` key.
2. User navigates **to button, then link inside, then next button, then link inside**, etc.

    https://github.com/user-attachments/assets/40e97745-5013-40c5-9070-a13e28fadf8f

3. **User triggers button, modal closes.** User triggers link, website opens.

    https://github.com/user-attachments/assets/496e8d7e-cabe-4ffa-8283-9dc8f4191f76

### 2. Hover State Makes Logo Unclear

The hover state should **highlight element, not obscure it**.

* User should see better what they point at, not see better what they do not point at (other logos).
* Partner would probably prefer their logo be highlighted when user shows interest, not have it be downplayed.

https://github.com/user-attachments/assets/b638bf28-9777-4c00-b544-738a8c6aa21e

### 3. No Hover State for Pentagon Pieces

1. User hovers over pentagon piece in the Wheel.
2. Pentagon **piece is not highlighted**.

https://github.com/user-attachments/assets/99343533-05f3-4bfa-8d92-c50a5e5726b5

> [!note]
> Even though cursor changes to pointer, when user moves point to another pentagon piece, the cursor remains a pointer, so it is unclear whether the entire pentagon is interactive, or just each piece.

**Comment by jeevangelista** [comment]
hover events should now be fixed.

**Comment by wesleyboar** [comment]
Confirmed. Tested.
